.github/workflows/cirrus-ci_retrospective.yml

```diff
@@ -0,0 +1,136 @@
+---
+
+# Notice when cirrus-ci has completed executing based on a push to
+# a Branch or Tag.  Prepare and automatically trigger the
+# Cirrus-CI build's 'manual' tasks as specified below.
+
+on:
+    # Ref: https://help.github.com/en/actions/reference/events-that-trigger-workflows#check-suite-event-check_suite
+    check_suite:  # Always/only triggered from the default branch (master)
+        types:
+            - completed
+
+# Variables required by multiple jobs/steps
+env:
+    # Version number of cirrus-ci_retrospective container image to use
+    CCI_RET_TAG: 'v1.1.3'
+    # Enable debugging of github actions itself
+    # (see https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-debug-message)
+    ACTIONS_STEP_DEBUG: '${{ secrets.ACTIONS_STEP_DEBUG }}'
+    MANUAL_TASK_NAME: "retrospective"
+    # Allow cevich to enable/disable failure notification
+    FAILMAILGIST: 'https://gist.githubusercontent.com/cevich/1e479c6811dc5b44508c2ba4dc295ff1'
+
+jobs:
+    # Obtain task details and validate required execution conditions
+    cirrus-ci_retrospective:
+        # Do not execute for other github applications, only works with cirrus-ci
+        if: github.event.check_suite.app.name == 'Cirrus CI'
+        runs-on: ubuntu-latest
+        steps:
+            # Ref: https://github.com/containers/automation/tree/master/cirrus-ci_retrospective
+            - name: Execute the cirrus-ci_retrospective container
+              uses: docker://quay.io/libpod/cirrus-ci_retrospective:${{env.CCI_RET_TAG}}
+              env:
+                GITHUB_TOKEN: ${{ github.token }}
+
+            # Ref: https://help.github.com/en/actions/reference/workflow-commands-for-github-actions
+            - name: Load and process resulting JSON into output parameter values
```

Comment:
There's probably something very simple I'm missing. In the [should never happen](https://github.com/containers/automation_sandbox/runs/577297684?check_suite_focus=true#step:6:218) log that you refer to in the comments, I see the previous step ("Execute the cirrus-ci_retrospective container") run, but I don't see this one run. Could that run have happened before you added this section? What have I missed?

Reply:
Oh, that's the `debug` job (look on the left-side pane).  It does a duplicate (parallel) run of the same container, but splats out the results instead of trying to interpret them.

Reply:
If you think it would be more clear, I can merge the steps from the separate debug job.  I did it separate like this in case the primary job failed w/o any useful output, and to avoid adding more output into an already complex operation.

Reply:
okay, did this, it's all in-line now.